packages/trezor-user-env-link/src/index.ts

```diff
@@ -1 +1,2 @@
 export * from './websocket-client';
+export * from './controllerManager';
```

Comment:
I guess we are going to merge these two exports into 1 in the future. not needed to do it now